client/app/js/color_select.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import SelectField from 'material-ui/SelectField';
+import MenuItem from 'material-ui/MenuItem';
+import LabelIcon from 'material-ui/svg-icons/action/label';
+
+const error_style = {
+  color: "rgb(244, 67, 54)",
+  fontSize: 12
+};
+
+const colors = [
+  { color: "blank", code: "#ffffff" },
```

Comment:
I just found out that we already have folder named contants and an existing color.js. Can you please merge everything and put color.js and this code in a file under contants/ ?